frontend/src/components/IndeterminateCheckbox.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
This component was added when the admin UI had a different design and is not currently used. I kept it around because it actually took me a bit of time to figure out and it gives us a nice option for a checkbox in the future. I'm open to removing it if you fell it clutters the codebase too much (it is in the adhoc repos history at this point so wouldn't be too hard to restore it if we end up deleting it)